ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.cpp

```diff
@@ -224,8 +222,10 @@ void CreateLoadReport::runReport(const QStringList& selectedPhaseNames)
   for (const CodedValue& codedValue : activeValues)
   {
     setUtilityTraceOrconditionWithCodedValue(codedValue);
-    TaskWatcher task = m_utilityNetwork->trace(m_traceParameters);
-    m_tasks[task.taskId()] = codedValue.name();
+    m_utilityNetwork->traceAsync(m_traceParameters).then(this, [this, codedValue](QList<UtilityTraceResult*>)
+    {
+      createTraceCompletedConnection(codedValue.name());
```

Comment:
Sorry I missed this on the previous review, but this method doesn't make sense anymore. It doesn't create any connections, so it should be renamed to convey what it actually does now.

Reply:
Renamed to onTraceCompleted_().